assets/i18n/qqq.json

```diff
@@ -439,6 +439,7 @@
 	"I18N_PREFERENCES_USERNAME": "Text displayed in the preferences page. - Text shown at the left of the text entry where the user can change his username.\n{{Identical|Username}}",
 	"I18N_PREFERENCES_USERNAME_NOT_SELECTED": "Text displayed in the preferences page. - Text shown in the text entry for the username when there is no username assigned to the user.",
 	"I18N_PROFILE_NO_EXPLORATIONS": "Text displayed on the Profile page. - This message is shown on a user's profile when the user has zero created/edited explorations.",
+	"I18N_REDIRECT_TO_LOGIN": "Text displayed on a button clicking on which redirects to login page",
```

Comment:
Text displayed on a button that, when clicked, allows the user to continue the signup process.

Reply:
Done.